locales/it_IT/pcsx2_Main.po

```diff
@@ -2565,64 +2557,64 @@ msgstr "Salvataggi di Stato:"
 
 #: pcsx2/gui/Panels/PathsPanel.cpp:39
 msgid "Select folder for Savestates"
-msgstr "Seleziona una cartella per i Salvataggi di Stato:"
+msgstr "Seleziona una cartella per i Salvataggi di Stato"
 
 #: pcsx2/gui/Panels/PathsPanel.cpp:46
 msgid "Snapshots:"
 msgstr "Snapshot:"
 
 #: pcsx2/gui/Panels/PathsPanel.cpp:47
 msgid "Select a folder for Snapshots"
-msgstr "Seleziona una cartella per le Snapshot:"
+msgstr "Seleziona una cartella per le Snapshot"
```

Comment:
Secondo te non sarebbe meglio screenshot?
O istantanee semmai in italiano.

Reply:
Me lo sono chiesto anch'io, ma ho scelto di mantenere snapshot perché è il nome di default che PCSX2 da alla cartella in questione. Non volevo creare confusione. Pensi che questa motivazione sia valida o mi sto facendo troppi problemi?
Ammetto che anch'io penso che screenshot sia meglio, ormai è di uso comune.

Reply:
Beh, però da salvataggio di stato a sstates diciamo che già lì butti all'aria l'immediatezza del nome. 
Sicchè.

Reply:
Whatever, I don't care anymore.
Va bene Screenshot.